src/app/src/utils/index.js

```diff
@@ -21,13 +40,45 @@ export class DataIndexer {
 
     // Return list of responses to a single question for current regions
     getResponse(questionKey) {
-        return this.geographies.map(geo => {
-            const idx = this.survey[questionKey].idx;
-            const d = this.data[geo];
-            const c = d['Combined'][idx];
-            const m = d['Male'][idx];
-            const f = d['Female'][idx];
-            return [{ geo: geo, combined: c, female: f, male: m }];
-        });
+        if (isQuestionCode(questionKey)) {
+            // These are question/response values for a single stacked question
+            const items = getAllResponsesForQ(questionKey);
+            return this.geographies.map(geo => {
+                const responses = items.map(key => this.formatForViz(key, geo));
+                const questionKey = responses[0].key;
+
+                return {
+                    question: questionFromResponse(this.survey[questionKey]),
+                    response: responses,
```

Comment:
I think this line should be 'responses', to match line 61 below.

Reply:
Thanks. I've unified `response/response` and `responses/responses` to better reflect when there is an expectation of a single response object or a list of them in c9b6961